web-common/src/features/dashboards/dimension-table/dimension-table-utils.ts

```diff
@@ -379,55 +392,67 @@ export function prepareDimensionTableRows(
 ): DimensionTableRow[] {
   if (!queryRows || !queryRows.length) return [];
 
-  const formattersForMeasures = Object.fromEntries(
-    allMeasuresForSpec.map((m) => [m.name, createMeasureValueFormatter(m)])
-  );
-
-  const tableRows: DimensionTableRow[] = queryRows.map((row) => {
-    if (!row.measureValues) return {};
-
-    const rawVals: [string, number][] = row.measureValues.map((m) => {
-      return [m.measureName ?? "", m.baseValue ? (m.baseValue as number) : 0];
-    });
-
-    const formattedVals: [string, string | number][] = rawVals.map(
-      ([name, val]) => ["__formatted_" + name, formattersForMeasures[name](val)]
+  const formattersForMeasures: { [key: string]: (val: number) => string } =
```

Comment:
there are no actual changes here, I just added types and prettier changed the formatting in a way that cascaded down the page.